src/lua/fiber.c

```diff
@@ -998,6 +1013,7 @@ static const struct luaL_Reg fiberlib[] = {
 	{"name", lbox_fiber_name},
 	{"check_slice", lbox_check_slice},
 	{"set_max_slice", lbox_fiber_set_max_slice},
+	{"get_max_slice", lbox_fiber_get_max_slice},
```

Comment:
Please refrain from adding new public functions. Instead of introducing this function for compat, better turn it into a tweak bool flag available in C.

Reply:
Discussed f2f and decided to duplicate default max slice value in `fiber.lua` given we only need it for compat option which will eventually will be dropped.